server/routes/userRoute.js

```diff
@@ -93,17 +92,29 @@ router.put("/:id", authorize, async (req, res, next) => {
   }
 })
 
-router.put("/picUpload/:id",
-  (req, res, next) => {
-    const image_upload = upload.single('image');
-    image_upload(req, res, error => {
-      if (error) {
-        res.status(422).json({ "Error": error.message })
-      }
-      // Save new url 
-      addPic(req.file.location, req.params.id);
-      res.status(200).send('Picture updated!');
-    })
-  });
+router.put("/pictures/:id", (req, res, next) => {
```

Comment:
pictures? what is this pictures for? is it for users? then it should be about putting users

you could do something like users/:id/avatars